go.mod

```diff
@@ -259,3 +259,4 @@ require (
 )
 
 replace github.com/csimplestring/delta-go => github.com/treeverse/delta-go v0.0.0-20250325160917-8c0ebb032f43
+replace github.com/golang-jwt/jwt => github.com/golang-jwt/jwt/v5 v5.2.2
```

Comment:
Why do we need a `replace` directive?  These are awful - for instance, they make it impossible to use packages from this module in another module _unless_ the same directive is used there.  That's also non-composable.

I would expect to remove `.../jwt/v3`, maybe `.../jwt/v4` from _somewhere_ in this go.mod, and not to use such a directive.

Reply:
Okay.  We should consider using a more maintained version, or another tool, or sending our own PR, or forking the original.  I do _not_ want to keep a `replace` directive in go.mod, they cause an intense amount of pain.